uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
-<html>
-<head>
-    <title>{{ c.EVENT_NAME }} - At the Door Registration</title>
-    <link rel="stylesheet" type="text/css" href="../static/styles/styles.css" />
-    <script type="text/javascript" src="../static/lib/jquery/jquery-1.7.2.min.js"></script>
-    <script src="../common.js" type="text/javascript"></script>
-    <script type="text/javascript">
-        var DISABLE_STRIPE_BUTTONS_ON_CLICK = true;
-        $(function(){
-            $(".focus:first").focus();
-            if (DISABLE_STRIPE_BUTTONS_ON_CLICK) {
-                $("form.stripe").on("click", "button", function() {
-                    $("a > .stripe-button-el").unwrap().attr("disabled", true).unwrap()
-                });
-            }
-        });
-    </script>
-</head>
-<body>
+{% extends "./preregistration/preregbase.html" %}
+{% block title %}At the Door Registration{% endblock %}
+{% block backlink %}{% endblock %}
+{% block message %}{% endblock %}
+{% block content %}
 
-<span id="message" style="float:left ; color:red">{{ message }}</span> <br/>
+<div class="masthead"></div>
+
+<script>
+    window.setTimeout(toastr.clear, 10000);
 
-<script src="../static/js/servertimecheck.js" type="text/javascript"></script>
-<script type="text/javascript">
-    $(function(){
-        $("#message").css("float", null);
-    });
-    window.setTimeout(function(){ $("#message").hide(); }, 10000);
-    
     var maybeBold = function() {
-        if ($("[name=payment_method]").val() == {{ c.MANUAL }}) {
-            $("#email").css("font-weight", "bold");
+        if ($.val('payment_method') === {{ c.MANUAL }}) {
+            $('#email').css('font-weight', 'bold');
         } else {
-            $("#email").css("font-weight", "normal");
+            $('#email').css('font-weight', 'normal');
         }
     };
     $(maybeBold);
-    
-    var showOrHide = function() {
-        setVisible("#underage", $("[name=age_group]").val() == {{ c.UNDER_18 }});
-        setVisible("#under13", $("[name=age_group]").val() == {{ c.UNDER_13 }});
-    };
-    {% if not c.COLLECT_EXACT_BIRTHDATE %} $(showOrHide); {% endif %}
+
+    $(function () {
```

Comment:
This next line is where we remove the kick-in rows.

The line after that moves the "require fields are in bold" message to be above the extra fields we added.